src/globals/brand-colours.ts

```diff
@@ -0,0 +1,14 @@
+export type brandColours =
```

Comment:
not a blocker but could we rename this to singular `brandColour` ? Types are generally singular, `number`, `string`, `Date` etc. If not I'll probably rename it later. Not a blocker.